tests/test_pods/bzip2.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Copyright © 2019 SUSE LLC
```

Comment:
not sure if failures in this script are expected but I would recommend to call  in here:

ˋˋˋ
set -euo pipefail
ˋˋˋ

Reply:
Added `set -o pipefail` to openqa_lib.sh initialization. `set -e` doesn't make sense because errors are handled by `trap qa_fail ERR`. `set -u` may be useful in other scripts but there's not much use for it here.